libs/helpers/helpers.go

```diff
@@ -0,0 +1,81 @@
+package helpers
+
+import (
+	"log"
+	"strconv"
+	"strings"
+)
+
+// LatestVersion gets a slice of versions and returns the latest version as a string
+// e.g. ["0.1", "0.10", "1.0"] or ["0.1.0", "0.10.1", "1.0.2"]
+// any number of minor/patch versions will work as long as all versions are consistent
+// i.e. this is not allowed ["0.1", "0.10.0", "1.0"] and the function will return ""
+func LatestVersion(vl []string) string {
```

Comment:
probably should return error too so you can bail from your errors below more gracefully